cfme/tests/configure/test_log_depot_operation.py

```diff
@@ -142,10 +143,11 @@ def configured_depot(log_depot, depot_machine_ip, appliance):
     server_log_depot = appliance.server.collect_logs
     with update(server_log_depot):
         server_log_depot.depot_type = log_depot.protocol
-        server_log_depot.depot_name = fauxfactory.gen_alphanumeric()
-        server_log_depot.uri = uri
-        server_log_depot.username = log_depot.credentials.username
-        server_log_depot.password = log_depot.credentials.password
+        if log_depot.protocol != 'dropbox':
```

Comment:
After some discussion about the dropbox selection and appliance versions, I think here there needs to be an additional check + pytest.skip when the protocol is dropbox and the appliance version is upstream.